gcp_variant_transforms/beam_io/vcfio_test.py

```diff
@@ -445,6 +448,35 @@ def test_info_numbers_and_types(self):
     self.assertEqual(2, len(read_data))
     self._assert_variants_equal([variant_1, variant_2], read_data)
 
+  def test_use_of_representative_header(self):
```

Comment:
Could you please add a test containing multiple files (sample1 defined in one file, sample2 in another). This tests that we don't end up with the same sample in all files.